qiskit_ibm_runtime/utils/validations.py

```diff
@@ -102,6 +103,18 @@ def validate_isa_circuits(circuits: Sequence[QuantumCircuit], target: Target) ->
             )
 
 
+def validate_rzz_pubs(pubs: Union[List[EstimatorPub], List[SamplerPub]]) -> None:
+    """Validate that rzz angles are always in the range [0, pi/2]
+
+    Args:
+        pubs: A list of pubs.
+    """
+    for pub in pubs:
+        message = is_rzz_pub(pub)
```

Comment:
Minor, but should it be called `is_not_rzz_pub`? It returns something True-like when the pub does not have rzz.

Reply:
I'll rename to `is_valid_rzz_pub`, it's not about whether there's `rzz` inside but about whether all the `rzz` are valid, I hope that the new name will be clear...

Reply:
Done in e98bcc13